### DISCUSSION TITLE
output file is larger than specified -size argument

### DISCUSSION THREAD

**Discussion by MaBecker** [ImageMagick/ImageMagick#4437] [Help] [answered]
This is the command:

```bash
convert -size 1024x625 -background white -fill black -font ArialMT.ttf -gravity north -pointsize 80 \
label:'Akku Set for - Skilled\nWorker'  xc: -gravity center -pointsize 240 -annotate +0+220 '10' \
-depth 8 gray:s3_1024x625x8.raw

ls -l 

-rw-r--r-- 1 root root 1280000 Nov  3 21:52 s3_1024x625x8.raw
```

expected a output size of 1024x625 : 640.000 byte and not the double size of 1.280.000 byte

Any hints how to get the expected file size?

**Comment by snibgo** [comment]
Your command creates two images.

"-size 1024x625" means a width of 1024 pixels and a height of 625 pixels. Filesize in bytes may be entirely different.

But with 1 byte per pixel, 1024x625 pixels per image, and two images, we get 1280000 bytes.

**Comment by MaBecker** [comment]
Hmm ... just changed the out put to `PNG:s3_1024x625x8.png` and got two files `s3_1024x625x8-0.png`  contains label and annotation and `s3_1024x625x8-1.png` only the annotation.  Was not  expecting two images. Is there a possibility to just write one?

s3_1024x625x8-0.png with both information and s3_1024x625x8-1.png only with annotation

![s3_1024x625x8-0](https://user-images.githubusercontent.com/15932646/140196053-a475eb7a-a8f6-4b31-b7dc-7c84da948d90.png)
![s3_1024x625x8-1](https://user-images.githubusercontent.com/15932646/140196071-64a4cac2-972b-46c8-81e1-fa2a137429cd.png)

**Comment by snibgo** [comment]
Your command creates two images. The first is from "label:"Akku Set for - Skilled\nWorker". The second is from "xc:" which is then annotated.

If you remove "xc:" then only one image will be created.

**Comment by MaBecker** [comment, marked as answer]
Hmm, can't remember why I added `xc:` .....

Great, many thanks!
